### ISSUE TITLE
Doesn't work with custom cargo-features in Cargo.toml 

### ISSUE THREAD

**Issue by aminya** [LukeMathWalker/cargo-chef#173]
When running `cargo chef cook --recipe-path recipe.json`, I cannot use cargo-features like `profile-rustflags`

I get this error:
```rs
error: config profile `release` is not valid (defined in `/home/test/.cargo/config.toml`)

Caused by:
  feature `profile-rustflags` is required

  The package requires the Cargo feature called `profile-rustflags`, but that feature is not stabilized in this version of Cargo (1.66.0-nightly (b332991a5 2022-10-13)).
  Consider adding `cargo-features = ["profile-rustflags"]` to the top of Cargo.toml (above the [package] table) to tell Cargo you are opting in to use this unstable feature.
  See https://doc.rust-lang.org/nightly/cargo/reference/unstable.html#profile-rustflags-option for more information about the status of this feature.
thread 'main' panicked at 'Exited with status code: 101', /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/cargo-chef-0.1.48/src/recipe.rs:176:27
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
#30 ERROR: process "/bin/sh -c cargo chef cook --recipe-path recipe.json" did not complete successfully: exit code: 101
```

It works with the standard `cargo build` command because the feature is specified in the `Cargo.toml` of the workspace.

**Comment by aminya** [comment]
Here you go:
https://github.com/aminya/cargo-chef-bug

And the error reproduction:
https://github.com/aminya/cargo-chef-bug/actions/runs/3672245178/jobs/6208245400#step:3:63

**Comment by LukeMathWalker** [comment]
This has been fixed and released as part of 0.1.50 👌🏻
